### PR TITLE
[RND201-203] output log to a file test

### PR DESCRIPTION
**Pull request by Anna-Majda-Mobica**

### CODE REVIEW

File: tests/tests/optionsTests.robot [hunk 1]
```diff
@@ -7,13 +7,9 @@ Resource            src/page_objects/keywords/common.robot
 Resource            src/page_objects/keywords/home_page.robot
 Resource            src/page_objects/keywords/ubuntu.robot
 Resource            src/test_files/test_files.resource
+Resource            __init__.robot
```
Comment (krzysztof-rodzaj-mobica) [inline tests/tests/optionsTests.robot:10]: Is that resource necessary?

File: tests/tests/optionsTests.robot [hunk 2]
```diff
@@ -87,4 +83,21 @@ Find and open Collision resolution option
...
+    Check If Log Console Contains    Opening "${file_4mb}[name]"
+    Validate Output Description    ${file_4mb}[archive_name]
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${LogsButton}
+    Click Element    ${OutputLogsSwitch}
+    Wait Until Element Is Visible    ${DownloadLogsButton}
+    Click Load Button
+    Click Element    ${DownloadLogsButton}
+    Switch Window    new
+    Wait Until Element Is Visible     //pre[contains(text(),'info: Opening "file_4MB.exe"')]
```
Comment (krzysztof-rodzaj-mobica) [inline tests/tests/optionsTests.robot:128]: this locator could be in locators file and used here as variable

File: tests/tests/optionsTests.robot (hunk 1 above)
Comment (Anna-Majda-Mobica) [reply tests/tests/optionsTests.robot:10]: done

File: tests/tests/optionsTests.robot (hunk 2 above)
Comment (Anna-Majda-Mobica) [reply tests/tests/optionsTests.robot:128]: done

**Comment by Anna-Majda-Mobica** [comment]
Due to local branch name change, I experienced some problems with pushing my latest changes into remote branch. Thus, I created a new change to review with a new, proper name  
https://github.com/Mobica/innoextract-wasm/pull/93

**Pull request closed without merging**